src/app/legacy/containers/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -485,6 +485,10 @@ exports[`FooterContainer snapshots should render correctly 1`] = `
         </ul>
         <p
           class="emotion-50 emotion-51"
+          />
+        </ul>
```

Comment:
This looks a bit funny 🤔

Reply:
Merge conflicts locally had:
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/72317734/233967467-8b7101b2-a53f-44c1-8cdf-c497968a8706.png">

I accepted both changes here as otherwise it would have removed the class for the `focusIndicatorInvert` and that is not what we want - any further thoughts?

Reply:
I would maybe delete the snapshot locally and re-run the test suite for that component to regenerate the snapshot based on all the code changes in the branch. That should generate a new snapshot with your changes and `latest` applied.

Reply:
I'll do the same for `src/app/Layouts/__snapshots__/defaultPageWrapper.test.jsx.snap` as a similar diff also exists there. I've just pulled from `origin`, deleted the snaps and am re-running w/ updates

Reply:
Just pushed updates now, if you wouldn't mind taking another quick peek whilst the GH actions run that would be great.

Reply:
Looks good 👍